frontend/src/pages/featureStore/FeatureStoreRoutes.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
+import { Route, Routes } from 'react-router-dom';
+import FeatureStore from './screens/FeatureStore';
+
+const FeatureStoreRoutes: React.FC = () => (
+  <Routes>
+    <Route path="/overview" element={<FeatureStore />} />
```

Comment:
As we have merged this [PR](https://github.com/ppadti/odh-dashboard/pull/5) update this to include FeatureStoreCoreLoader file to consume feature store project context